app/views/publishing_users/passwords/new.html.erb

```diff
@@ -1,16 +1,24 @@
-<h2>Forgot your password?</h2>
+<main id="content" role="main" tabindex="-1">
```

Comment:
copy on alpha is: 
# Password Reset
Forgotten your password?
Enter your email address below, and we'll email instructions for setting a new one.